charts/chronograf/README.md

```diff
@@ -84,35 +76,39 @@ The following table lists the configurable parameters of the chronograf chart an
 | `tolerations`                | Toleration labels for pod assignment                                                                      | []                                          |
 | `affinity`                   | Affinity settings for pod assignment                                                                      | {}                                          |
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+To configure the chart, do either of the following:
 
-```bash
-helm upgrade --install my-release \
-  --set ingress.enabled=true,ingress.hostname=chronograf.foobar.com \
-    influxdata/chronograf
-```
+- Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example, use the following command:
 
-The above command enables persistence and changes the size of the requested data volume to 200GB.
+  ```bash
+  helm upgrade --install my-release \
+    --set ingress.enabled=true,ingress.hostname=chronograf.foobar.com \
+      influxdata/chronograf
+  ```
 
-Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+  This command enables persistence and changes the size of the requested data volume to 200GB.
 
-```bash
-helm upgrade --install my-release -f values.yaml influxdata/chronograf
-```
+- Provide a YAML file that specifies parameter values while installing the chart. For example, use the following command:
+
+  ```bash
+  helm upgrade --install my-chart-name -f values.yaml influxdata/chronograf
+  ```
+
+  > **Tip**: Use the default [values.yaml](values.yaml).
 
-> **Tip**: You can use the default [values.yaml](values.yaml)
+For more information about running Chronograf in Docker, see the [full image documentation](https://quay.io/influxdb/chronograf).
```

Comment:
Have we been discussing how to run Chronograf in Docker? Or should we drop the word "more"?

Reply:
Can drop